extensions/wikia/AdEngine/js/spec/slot/resolvedState.spec.js

```diff
@@ -2,128 +2,140 @@
 describe('ext.wikia.adEngine.slot.resolvedState', function () {
 	'use strict';
 
-	var BIG_IMAGE = 'bigImage.png',
+	function createCorrectParams() {
+		return {
+			imageSrc: BIG_IMAGE,
+			aspectRatio: ASPECT_RATIO,
+			resolvedState: {
+				aspectRatio: RESOLVED_STATE_ASPECT_RATIO,
+				imageSrc: RESOLVED_IMAGE
+			},
+			backgroundImage: {
+				src: DEFAULT_IMAGE
+			}
+		}
+	}
+
+	function createIncorrectParams() {
+		return {
+			aspectRatio: 1,
+			imageSrc: BIG_IMAGE,
+			resolvedState: {
+				aspectRatio: 0,
+				imageSrc: ''
+			},
+			backgroundImage: {
+				src: DEFAULT_IMAGE
+			}
+		}
+	}
+
+	function createCorrectParamsWithTwoAssets() {
+		return {
+			aspectRatio: ASPECT_RATIO,
+			resolvedStateAspectRatio: RESOLVED_STATE_ASPECT_RATIO,
+			image1: {
+				element: {
+					src: DEFAULT_IMAGE
+				},
+				defaultStateSrc: BIG_IMAGE,
+				resolvedStateSrc: RESOLVED_IMAGE
+			},
+			image2: {
+				element: {
+					src: DEFAULT_IMAGE
+				},
+				defaultStateSrc: BIG_IMAGE_2,
+				resolvedStateSrc: RESOLVED_IMAGE_2
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move the declaration of "RESOLVED_IMAGE_2" before this usage. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AVariableDeclarationAfterUsage)